geoshape/challenges/default.json

```diff
@@ -128,13 +128,216 @@
           }
         },
         {
-          "operation": "polygon",
+          "operation": "polygon-intersects",
+          "warmup-iterations": 200,
+          "iterations": 100,
+          "target-throughput": 3.3
```

Comment:
similar to @salvatore-campagna question, were these obtained from measurements done on hardware close to the nightly ones (https://elasticsearch-benchmarks.elastic.co/#)?

Reply:
I used the machine where I run the lucene benchmarks so it is probably pretty close.

Reply:
@iverase I have no idea which machines are those. Anyway, given recent developments about the test methodology, maybe we can try to proceed without having `target-throughput` or `target-interval`? That would ideally make these tests not hardware-dependent?